### ISSUE TITLE
Image Descriptions

### ISSUE THREAD

**Issue by qwertynerd97** [ucfopen/Obojobo#653]
We provide an alt text field for our images, but users cannot currently add an image description unless they put it in the caption.  We may want to provide a separate field for image descriptions so that they can be toggled off by individual students. Also, image descriptions are usually displayed in brackets, so a special field could automatically provide the proper layout, instead of leaving it up to individual instructors. 
Standard format for image description: 
[**Image Description**: ...the content of the description...]

Here is [an article on the difference between alt text and image descriptions](http://www.perkinselearning.org/technology/blog/how-write-alt-text-and-image-descriptions-visually-impaired)
